locale/fi/LC_MESSAGES/django.po

```diff
@@ -3574,6 +3579,74 @@ msgstr "Ei osumia"
 msgid "Search for..."
 msgstr "Etsi..."
 
+#: templates/support_channels.html
+msgid "Documentation"
+msgstr "Asiakirjat"
+
+#: templates/support_channels.html
+msgid "Aplus manual"
+msgstr "Aplus-käsikirja"
+
+#: templates/support_channels.html
+msgid "Aplus website"
+msgstr "Aplus-verkkosivusto"
+
+#: templates/support_channels.html
+msgid "Community"
+msgstr "Yhteisö"
+
+#: templates/support_channels.html
+msgid "Microsoft Teams"
+msgstr "Microsoft Teams"
+
+#: templates/support_channels.html
+msgid "Create an issue in Github"
+msgstr "Luo ongelma Githubissa"
+
+#: templates/support_channels.html
+msgid "Announcements"
+msgstr "Ilmoitukset"
+
+#: templates/support_channels.html
+msgid "Current Development"
+msgstr "Nykyinen kehitys"
+
+#: templates/support_channels.html
+msgid "Other servers that use A+"
```

Comment:
We should not hardcode the name "A+" since local_settings.py may override the brand name of the site. Thus, the strings need a variable for the brand name.